src/components/core/navbar/index.tsx

```diff
@@ -198,10 +197,10 @@ const Navbar = ({ starCount }: { starCount?: number }) => {
 
         {/* Mobile menu */}
         {isMenuOpen && (
-          <div className="bg-secondary-light bg-opacity-25 dark:bg-primary-light dark:bg-opacity-25 h-full z-10 w-1/2 fixed inset-y-0 right-0 md:hidden transition-transform duration-300">
+          <div className="bg-secondary bg-opacity-90 bg-gradient-to-b dark:from-primary-light dark:to-primary-light/30 dark:bg-opacity-90 h-full fixed right-0 inset-y-0 w-[70%] md:hidden transition-transform duration-300">
```

Comment:
Opacity 90 would be too dark considering the pink shade, can we go lighter? around 60 or 70. Also give top-0 as well